doc/develop/extending.rst

```diff
@@ -193,46 +193,51 @@ of wxPython, which must be 3.x, and OpenCV, which must be 2.4.
     the Odemis virtualenv.
 #.  Install wheel, so we can install binary packages using pip:
     ``pip install wheel``.
-#.  Install futures using ``pip install futures``
```

Comment:
From https://pypi.org/project/futures/:

This is a backport of the concurrent.futures standard library module to Python 2.

It does not work on Python 3 due to Python 2 syntax being used in the codebase. Python 3 users should not attempt to install it, since the package is already included in the standard library.